### PR TITLE
Search movies by title

### PR DESCRIPTION
**Pull request by mobolajii**

### CODE REVIEW

**Comment by laila-sb** [comment]
As discussed in slack the two tables need to be merged into one, as team decided to go forward with one table

**Review by mehtapparkinson — CHANGES_REQUESTED** [review]
when movie name exists in the DB, it returns the existing movie successfully. However, when searched for nonexistent movie it only logs "movie not found" in the backend. I would suggest returning a response entity with 404 and a not found message

File: README.md [hunk 1]
```diff
@@ -1,106 +1,2 @@
...
-- GitHub Actions will be used for continuous integration, with stages for building, testing, and deploying the application.
-
-2. **Docker**:
-- The application will be containerized using Docker. A Dockerfile and a `docker-compose.yaml` will be provided to run the application along with a MySQL container.
-
-3. **Pipeline**:
-- **Stage 1**: Build and test the application.
-- **Stage 2**: Build Docker image.
-- **Stage 3**: Deploy the application using Docker Compose.
+# java-group-project
```
Comment (mehtapparkinson) [inline README.md:1]: is it possible to take this file out of the PR?

File: src/main/java/com/cfg_java_team_jason/java_group_project/controller/MovieController.java [hunk 2]
```diff
@@ -21,82 +21,62 @@ public class MovieController {
...
-            }
-        //200 OK
-        logger.info("Successfully retrieved {} movies from the database.", movies.size());
-        return ResponseEntity.ok(movies);
-        } catch (Exception e) {
-            //500 Error
-            logger.error("An error occurred while retrieving movies from the database.", e);
-            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
-        }
+    public List<Movie> getAllMovies(){
```
Comment (mehtapparkinson) [inline src/main/java/com/cfg_java_team_jason/java_group_project/controller/MovieController.java:24]: getAll movies seems to be reverted to previous version here, can we undo that?

**Review by mehtapparkinson — CHANGES_REQUESTED** [review]
It seems like there was an issue when you were pulling the changes from development into your branch, which caused a lot of deletions. Could you review and undo those changes to ensure everything is correctly synced with development?

**Pull request closed without merging**